src/Microsoft.VisualStudio.ProjectSystem.Managed/Tree/Dependencies/IDependenciesChanges.cs

```diff
@@ -4,6 +4,8 @@
 
 #nullable enable    
 
+#nullable disable
+
```

Comment:
Actually is this correct?

Reply:
GitHub's choking on this PR. Will fetch locally and re-check.

Reply:
That looks like the only one, the extra whitespace after enable threw off the RegEx.